sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/AnalyzeSentimentWithOpinionMining.java

```diff
@@ -0,0 +1,95 @@
+
+import static com.azure.ai.textanalytics.models.TextSentiment.MIXED;
+import static com.azure.ai.textanalytics.models.TextSentiment.NEGATIVE;
+import static com.azure.ai.textanalytics.models.TextSentiment.POSITIVE;
+
+/**
+ * Sample demonstrates how to synchronously analyze the sentiment of document with opinion mining.
+ */
+public class AnalyzeSentimentWithOpinionMining {
+    /**
+     * Main method to invoke this demo about how to analyze the sentiment of document.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        // Instantiate a client that will be used to call the service.
+        TextAnalyticsClient client = new TextAnalyticsClientBuilder()
+            .credential(new AzureKeyCredential("{key}"))
+            .endpoint("{endpoint}")
+            .buildClient();
+
+        // The text that needs be analyzed.
+        String document = "Bad atmosphere. Not close to plenty of restaurants, hotels, and transit! Staff are not friendly and helpful.";
+
+        System.out.printf("Text = %s%n", document);
+
+        AnalyzeSentimentOptions options = new AnalyzeSentimentOptions().setIncludeOpinionMining(true);
+        final DocumentSentiment documentSentiment = client.analyzeSentiment(document, "en", options);
+        SentimentConfidenceScores scores = documentSentiment.getConfidenceScores();
+        System.out.printf(
+            "Recognized document sentiment: %s, positive score: %f, neutral score: %f, negative score: %f.%n",
+            documentSentiment.getSentiment(), scores.getPositive(), scores.getNeutral(), scores.getNegative());
+
+        List<MinedOpinion> positiveMinedOpinions = new ArrayList<>();
+        List<MinedOpinion> mixedMinedOpinions = new ArrayList<>();
+        List<MinedOpinion> negativeMinedOpinions = new ArrayList<>();
+        documentSentiment.getSentences().forEach(sentenceSentiment -> {
+            SentimentConfidenceScores sentenceScores = sentenceSentiment.getConfidenceScores();
+            System.out.printf("\tSentence sentiment: %s, positive score: %f, neutral score: %f, negative score: %f.%n",
+                sentenceSentiment.getSentiment(), sentenceScores.getPositive(), sentenceScores.getNeutral(), sentenceScores.getNegative());
+            sentenceSentiment.getMinedOpinions().forEach(minedOpinion -> {
+                TextSentiment aspectTextSentiment = minedOpinion.getAspect().getSentiment();
+                if (NEGATIVE.equals(aspectTextSentiment)) {
+                    negativeMinedOpinions.add(minedOpinion);
+                } else if (POSITIVE.equals(aspectTextSentiment)) {
+                    positiveMinedOpinions.add(minedOpinion);
+                } else if (MIXED.equals(aspectTextSentiment)) {
+                    mixedMinedOpinions.add(minedOpinion);
+                }
+            });
+        });
+
+        System.out.printf("Positive aspects count: %d%n", positiveMinedOpinions.size());
```

Comment:
Are we only creating the opinion list to print out the size ?
Seems like not adding a lot of value. 
This example could be simplified I think.

Reply:
We are starting with printing out the size. The for loop, it loops all mined opinions.

Reply:
ignore my last comment.  Changed the sample scenarios already.